common/docs/ANNOUNCEMENT.md

```diff
@@ -43,19 +43,21 @@ pluginManagement {
 ```
 _(this step will be redundant once this plugin is published to the Gradle Plugin Portal)._
 
-After that, we can configure the image build by using a `nativeBuild` configuration block:
+After that, we can configure the image build by using a `graal` configuration block:
 ```groovy
-nativeBuild {
-  imageName = "my-app"
-  mainClass = "org.test.Main"
-  verbose = true
-  fallback = false
+javaNative {
+    images {
+        main {
+            imageName = "my-app"
+            mainClass = "org.test.Main"
+            verbose = true
+            fallback = false
```

Comment:
Can we make this the default?

Reply:
Can you clarify what default you're talking about? The defaults are configured at https://github.com/melix/native-build-tools/blob/40c44217afd8b5e9f0e272190740c0f331ee8140/native-gradle-plugin/src/main/java/org/graalvm/buildtools/gradle/internal/BaseNativeImageOptions.java#L218-L231

You'd want `verbose` to be true by default?

Reply:
Looks like fallback is already false by default so we are good here